README.md

```diff
@@ -91,112 +128,153 @@ To compile this plugin, you need:
+    export SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN="GPU/P"
 
 ### Environment variables
 
-* `SCOREP_X86ENERGY_INTERVAL_US`/`VT_X86ENERGY_INTERVAL_US` (default=100000)
+* `SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN`/`SCOREP_METRIC_X86_ENERGY_PLUGIN`
 
-    The time in usecs, between two consecutive reads of the power/energy values. A longer interval
-    means less disturbance, a shorter interval is more exact.
+    Comma-separated list of sensors, and physical quantity. Can also contain wildcards, e.g.
+    `BLADE/E,*/E`
 
-    On Intel CPUs, the registers are updated roughly every msec. If you choose an interval of 1ms
-    you might find highly variating power consumptions. To gain most exact values, you should set
-    the amplitude to 10, if you can live with less accuracy, you should set it to 100000.
+* `SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN_VERBOSE`/`SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN_VERBOSE` (default `WARN`)
 
-* `SCOREP_X86ENERGY_CPU_PIN`/`VT_X86ENERGY_CPU_PIN` (default=no pinning)
+    Controls the output verbosity of the plugin. Possible values are: `FATAL`, `ERROR`, `WARN`,
+    `INFO`, `DEBUG`. If set to any other value, `DEBUG` is used. Case in-sensitive.
 
-    The CPU to pin the register reading thread to. The reading thread uses the `msr` files of core
-    `<X86_ENERGY_PLUGIN_CPU_PIN>` of each package (default: the first core of a socket). You might
-    consider to set it to a free Hyperthread.
+* `SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN_OFFSET` (default `70000.0`)
 
-* `SCOREP_X86ENERGY_BUF_SIZE`/`VT_X86ENERGY_BUF_SIZE` (default=4M)
+    Can be set to any double variable. Defines the offset in mW that is added to 
+    the blade, because with RAPL only the energy consumption of the
+    CPU is measurable. Blade is the sum of all RAPL values excepted the core
+    because there are included in package.
+    The default value of 70000.0 mW was measured on Taurus
 
-    The size of the buffer for storing samples. Can be suffixed with G, M, and K.
+* `SCOREP_X86_ENERGY_PLUGIN_READING_TIME` (default `5`)
+   or like the old plugin: `SCOREP_X86_ENERGY_PLUGIN_INTERVALL_US`
 
-    The buffer size is per counter per package, e.g., on a system with 3 energy counters, 2 sockets
-    and 4 MB buffer size this would be 24 MB in total.
+    The time in millisecs, between two consecutive reads of the power/energy values. A longer interval
+    means less disturbance, a shorter interval is more exact.
 
-    Typically, a sample consists of a 8 byte timestamp and 8 byte per selected counter. If the
-    buffer is too small, it might not be capable of storing all events. If this is the case, then an
-    error message will be printed to `stderr`.
+    On Intel CPUs, the registers are updated roughly every msec. If you choose an interval of 1ms
+    you might find highly variating power consumptions. To gain most exact values, you should set
+    the amplitude to 1, if you can live with less accuracy, you should set it to 100.
```

Comment:
Okay, but this is the original text from rschoene as you can see in [here](https://github.com/score-p/scorep_plugin_x86_energy/tree/a11bbb12b5f3ee26fad8979420b5f31100620f38).